packages/runtime-common/tests/query-test.ts

```diff
@@ -1619,6 +1619,424 @@ const tests = Object.freeze({
       ]);
     }
   },
+
```

Comment:
Let’s add a test that can filter range thru a plural field:
Like `range: { friends.age: { gt: 3 } }` where friends is a linksToMany.

Reply:
Let’s also add a test where the field itself is a containsMany. Maybe like `lotteryNumbers` containsMany field, and you want to find all the people who choose lottery numbers above 50 (you have to add a new field to the host and server test cards)

Reply:
Added the tests [650053e](https://github.com/cardstack/boxel/pull/1194/commits/650053ed178b2c2bca48f9c125b81fadebfeb010)